### PR TITLE
Change db - now Heroku compatible

### PR DESCRIPTION
**Pull request by henryhobhouse**
Database changed from sqlite to postgres to allow upload to Heroku. Example can be found here:

https://radiant-beyond-71056.herokuapp.com/posts

Updated Readme.md to inform those that have installed locally that they will need re-initialise their local db setup.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -48,6 +48,8 @@ group :development do
   gem "rubocop", "0.48.1"
   gem "rubocop-rails"
   gem "web-console", ">= 3.3.0"
+
+
```
Comment (neoeno) [inline Gemfile]: Get rid of these extra blank lines and we'll be good to go!

**Comment by henryhobhouse** [comment]
will re-submit with code ammendments from rubocop/Kay

**Comment by neoeno** [comment]
Take a look at the build — think some updates might be required.

**Review by neoeno — CHANGES_REQUESTED** [review]
Take a look at the build — think some updates might be required.

**Review by neoeno — APPROVED** [review]
NICE

**Comment by neoeno** [comment]
(Well, after resolving the conflicts at least!)

**Comment by henryhobhouse** [comment]
@neoeno I've added the travis.yml and changed the config to use postgres with added script to create the DB's needed to run rspec. All working now.

**Pull request merged**